sprint 10

DanielMayakov opened the pull request.

DanielMayakov27 committed:
sprint 10

Nick-review commented on app/src/main/res/layout/track_item.xml:
```diff
@@ -0,0 +1,69 @@
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="8dp"
```
Вынести в dimens

Nick-review commented on app/src/main/java/com/example/playlistmaker/SearchActivity.kt:
```diff
@@ -7,28 +7,57 @@ import android.text.TextWatcher
     private val simpleTextWatcher = object : TextWatcher {
-
         override fun onTextChanged(s: CharSequence?, s1: Int, s2: Int, s3: Int) {
             if (s.isNullOrEmpty()) {
```
можно так searchBinding.clearImageView.isVisible = !s.isNullOrEmpty()

DanielMayakov27 committed:
sprint 10
fixed some bugs
updated all
refactored code
Надеюсь я все ошибки с цветами убрал. Есть некоторые проблемы с восприятием цветов на моём устройстве

DanielMayakov27 committed:
sprint 11
